babel/dates.py

```diff
@@ -18,7 +18,12 @@
 
 import re
 import warnings
-import pytz as _pytz
+
+try:
+    import zoneinfo
+except ModuleNotFoundError:
+    zoneinfo = None
+    import pytz as _pytz
```

Comment:
I think we should support both zoneinfo and pytz, just in case someone is relying on pytz behavior even on Python 3.9+ – they could just install `pytz` and Babel would prefer it and work like before?

While we're at it, could get rid of the `_pytz` alias:

```suggestion
try:
    import pytz
except ModuleNotFoundError:
    pytz = None

try:
    import zoneinfo
except ModuleNotFoundError:
    zoneinfo = None
```